### ISSUE TITLE
insufficient documentation: configuration options

### ISSUE THREAD

**Issue by FlorinAndrei** [nicehash/excavator#97]
In the README a few command line options are shown. But the miner has many more config items. These need to be described in the documentation too - what they are and what they do.

**Comment by Kayfolom** [comment]
Все параметры командной строки описаны в [](https://github.com/nicehash/excavator#cmdline) , остальное (пулы/воркеры/разгон и т.д.) управляется из командного json файла (или через API)

**Comment by FlorinAndrei** [comment]
If I understand you correctly (Russian is not my native language), the existing command-line parameters are already described, and the rest of the configuration is controlled via the command json file or the API.

Let me ask a slightly different question: is there a way at all to tell the miner to use a certain GPU (instead of all GPUs), via the command-line options? Not via config files, but via command line options.

I'm running excavator from a set of scripts. It's very easy to control it from command line parameters. Re-creating config files would be more complex, and perhaps not always doable (issues with directory ownership, etc). It would be a lot easier if I could control things like GPU selection from the command line directly - but I'm not sure if this is currently doable with this miner.

Full control via command line options is easy with other miners (such as Claymore, sgminer, etc), but excavator seems unique in that it's a little more difficult to operate this way.

**Comment by FlorinAndrei** [comment]
Nevermind, I've figured it out:

https://github.com/nicehash/excavator/issues/96

TLDR: excavator-nicehash uses the JSON file for almost all of its configuration. Don't bother using command line parameters. Just put everything in the JSON file - see the issue linked above.
